react/ProductDetails.js

```diff
@@ -225,19 +224,55 @@ class ProductDetails extends Component {
   getHighlights() {
```

Comment:
This implementation could be better in my opinion. 
You are declaring `let highlights` and have 3 functions that could generate a side effect on its value...
Would be better if you just make that functions pure, returning the value instead of changing `highlights`.